router.go

```diff
@@ -59,8 +59,8 @@ func (r *Router) Add(method, path string, h HandlerFunc) {
 	if path[0] != '/' {
 		path = "/" + path
 	}
-	ppath := path        // Pristine path
-	pnames := []string{} // Param names
+	pPath := path       // Pristine path
```

Comment:
I'm all for short variable declarations, but if we need to comment variable names, maybe we should at least consider using longer names? These names seems to be used away enough from where they were declared that they qualify for longer names. `paramNames`, etc.

Reply:
this is internal name.. and has comment

Reply:
We stick to short names as Go guidelines recommend - specially used internally.

Reply:
I'll revert this part